client/package.json

```diff
@@ -17,9 +17,12 @@
     "@tailwindcss/vite": "^4.1.5",
     "@tanstack/react-router": "^1.116.0",
     "@tanstack/react-router-devtools": "^1.116.0",
+    "axios": "^1.10.0",
+    "cheerio": "^1.1.0",
```

Comment:
Considering we don't need these in our production application, We can make our deployments slightly smaller and faster by moving these to devDependencies.
```suggestion
```